src/modules/colorPicker/ColorPickerUI/Behaviors/MoveWindowBehavior.cs

```diff
@@ -7,21 +11,19 @@ namespace ColorPicker.Behaviors
 {
     public class MoveWindowBehavior : Behavior<Window>
     {
-        public static DependencyProperty LeftProperty = DependencyProperty.Register("Left", typeof(double), typeof(MoveWindowBehavior), new PropertyMetadata(new PropertyChangedCallback(LeftPropertyChanged)));
+        private static readonly DependencyProperty LeftProperty = DependencyProperty.Register("Left", typeof(double), typeof(MoveWindowBehavior), new PropertyMetadata(new PropertyChangedCallback(LeftPropertyChanged)));
 
-        private static void LeftPropertyChanged(DependencyObject d,
-               DependencyPropertyChangedEventArgs e)
+        private static void LeftPropertyChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
         {
             var sender = ((MoveWindowBehavior)d).AssociatedObject;
             var move = new DoubleAnimation(sender.Left, (double)e.NewValue, new Duration(TimeSpan.FromMilliseconds(150)), FillBehavior.Stop);
             move.EasingFunction = new QuadraticEase() { EasingMode = EasingMode.EaseOut };
             sender.BeginAnimation(Window.LeftProperty, move, HandoffBehavior.SnapshotAndReplace);
         }
 
-        public static DependencyProperty TopProperty = DependencyProperty.Register("Top", typeof(double), typeof(MoveWindowBehavior), new PropertyMetadata(new PropertyChangedCallback(TopPropertyChanged)));
+        private static readonly DependencyProperty TopProperty = DependencyProperty.Register("Top", typeof(double), typeof(MoveWindowBehavior), new PropertyMetadata(new PropertyChangedCallback(TopPropertyChanged)));
 
-        private static void TopPropertyChanged(DependencyObject d,
-               DependencyPropertyChangedEventArgs e)
+        private static void TopPropertyChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
```

Comment:
same

Reply:
all that happened here was put stuff on same line